src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -297,6 +304,16 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
             )}
           </TableCell>
         )}
+        {!isArchived && (
+          <TableCell>
+            <IconButton
+              onClick={() => setOpenCollapsable(!openCollapsable)}
+              size="small"
+              data-testid={dataTestId.registrationWizard.files.expandFileRowButton}>
+              {openCollapsable ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+            </IconButton>
+          </TableCell>
+        )}
```

Comment:
Moved this open-arrow to the the same line instead of under

Reply:
Med denne endringen bør det legges på en bottom margin på den skjulte raden (på linje 328)
```
<Box
  sx={{
    m: '1rem', <----- Denne (tidligere "m: '1rem 1rem 0 1rem'")
    display: 'grid',
    gridTemplateColumns: '1fr auto',
    gap: '2rem',
  }}>
```